### PR TITLE
[JENKINS-40001] Add plugin description to POM.

### PR DESCRIPTION
**Pull request by varyvol**
[JENKINS-40001](https://issues.jenkins-ci.org/browse/JENKINS-40001)

@reviewbybees, specially @stephenc @olamy @kohsuke

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
   <name>Jenkins SSH Slaves plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/SSH+Slaves+plugin</url>
+  <description>Allows to manage slaves running on Unix machines over SSH</description>
```
Comment (rsandell) [inline pom.xml]: We don't call them slaves any more

Comment (rsandell) [inline pom.xml]: Also differs slightly from the existing description in readme.md

Comment (varyvol) [inline pom.xml]: I have changed the text to use launch instead of manage and also agents instead of slaves.

File: pom.xml [hunk 2]
```diff
@@ -13,6 +13,7 @@
 
   <name>Jenkins SSH Slaves plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/SSH+Slaves+plugin</url>
```
Comment (recena) [inline pom.xml]: As part of this PR, I'd change `http` to `https`

**Comment by varyvol** [comment]
@recena @jglick done.

**Comment by recena** [comment]
🐝

**Comment by varyvol** [comment]
@jglick @stephenc could you please review this?

File: pom.xml [hunk 3]
```diff
@@ -12,7 +12,8 @@
   <packaging>hpi</packaging>
 
   <name>Jenkins SSH Slaves plugin</name>
-  <url>http://wiki.jenkins-ci.org/display/JENKINS/SSH+Slaves+plugin</url>
+  <url>https://wiki.jenkins-ci.org/display/JENKINS/SSH+Slaves+plugin</url>
+  <description>Allows to launch agents running on Unix machines over SSH</description>
```
Comment (stephenc) [inline pom.xml]: append `using a Java implementation of the SSH protocol`

File: src/main/resources/index.jelly [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<?jelly escape-by-default='true'?>
+<div>
+  Allows to launch agents running on Unix machines over SSH.
```
Comment (stephenc) [inline src/main/resources/index.jelly]: ditto for change in pom

**Comment by varyvol** [comment]
@stephenc changes done.

**Comment by varyvol** [comment]
@reviewbybees done.

**Pull request merged**